Sources/KetchSDK/UI/PresentationItem/PresentationItem.swift

```diff
@@ -245,6 +245,12 @@ extension KetchUI.ExperienceOption {
 
         case .preferencesTab(let tab):
             return (key: "ketch_preferences_tab", tab.rawValue)
+            
+        case .preferencesTabs(let tabs):
+            return (key: "ketch_preferences_tabs", tabs)
+            
+        case .sdkEnvironment(let url):
```

Comment:
there is one param that is named as `environment` , to avoid confusion lets set this as `sdkEnvironmentUrl()`